doc/api/deprecations.md

```diff
@@ -2497,6 +2497,19 @@ Type: Runtime
 Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 `Promise` instead, or a listener to the worker’s `'exit'` event.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: ClientRequest.abort is destroy
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28683
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+`[ClientRequest.destroy()`][] should be the same as `[ClientRequest.abort()`][]. Make ClientRequest more streamlike by deprecating abort().
```

Comment:
As a heads up, the linter is going to complain about this not being wrapped at 80 characters